src/services/movies.ts

```diff
@@ -10,11 +12,9 @@ export class MoviesService {
         },]
     }
 
-    public static getOne() {
-        return {
-            id: 46,
-            title: "Avatar - Voie de l'eau"
-        }
+    public static async getOne() {
+        const oneMovie =  await prisma.movies.findFirst();
```

Comment:
Si ton instruction est simple, tu peux écrire directement
```suggestion
    return await prisma.movies.findFirst();
```